exp-timing/femto.py

```diff
@@ -58,124 +57,121 @@ def __init__(self, nx='NULL'):
 
         nm = 'XCS'
         namelist.add(nm)
-        base = 'LAS:FS4:'  # base name for this sysetm
-        dev_base[nm] = base+'VIT:'
+        base = 'LAS:FS4:'  # Base PV name for this laser locker.
+        dev_base[nm] = base+'VIT:' 
         matlab_pv_base[nm] = dev_base[nm]+'matlab:'
-        matlab_pv_offset[nm] = 1
+        matlab_pv_offset[nm] = 1 # Notepad PV numbering starts at 1.
         matlab_pv_digits[nm] = 2
-        counter_base[nm] = base+'CNT:TI:'   # time interval counter
+        counter_base[nm] = base+'CNT:TI:'   # PV name for the Time Interval Counter (SR620)
         freq_counter[nm] = dev_base[nm]+'FREQ_CUR'        
         phase_motor[nm] = base+'MMS:PH' 
         error_pv_name[nm] = dev_base[nm]+'FS_STATUS' 
         version_pv_name[nm] = dev_base[nm]+'FS_WATCHDOG.DESC' 
-        laser_trigger[nm] = 'EVR:LAS:XCS:01:TRIG0:TDES' # was 'LAS:SR63:EVR:09:CTRL.DG0D'
-        trig_in_ticks[nm] = 0  # experiment side triggers operate in ticks units
-        reverse_counter[nm] = 1
-        use_secondary_calibration[nm] = 0
+        laser_trigger[nm] = 'EVR:LAS:XCS:01:TRIG0:TDES' # Name of on time EVR trigger channel.
+        trig_in_ticks[nm] = 0  # The laser_trigger PV is the delay value in ns. 0 sets trigger value to ns.
```

Comment:
This is unclear. Are you saying that this is a value with ns units or that it's a bool where 0 sets trigger value to ns and 1 sets it to something else?

Reply:
The trig_in_ticks dictionary was removed so this comment is no longer present.